Assets/HoloToolkit/Sharing/Tests/ImportExportAnchorManager.cs

```diff
@@ -116,18 +116,19 @@ public bool AnchorEstablished
     /// </summary>
     RoomManagerAdapter roomManagerCallbacks;
 
-    void Start()
+    void Awake()
```

Comment:
Unfortunately, we found that FindObjectOfType will fail if Awake() hasn't been called yet. Since the execution order can vary, this was occasionally seen in BasicCursor.cs and raised in #246. 

In addition, @HodgsonSDAS brings up a good point. The new pattern has Instance set in Awake(), so there's no guarantee it will be set in any other Awake().